tools/deb/envoy_bootstrap_v2.json

```diff
@@ -2,21 +2,22 @@
   "node": {
     "id": "{{ .nodeID }}",
     "cluster": "{{ .cluster }}",
-{{ if .zone }}
+    {{ if .zone }}
```

Comment:
The JSON in this file was inconsistently structured (and in places invalid due to trailing commas) so I went ahead and updated this to make it easier to modify.